contracts/lib/MetaBlock.sol

```diff
@@ -22,6 +22,11 @@ library MetaBlock {
     bytes32 constant ORIGINTRANSITION_TYPEHASH = keccak256(
         "OriginTransition(uint256 dynasty,bytes32 blockHash,bytes20 coreIdentifier)"
     );
+    /** To hash vote message according to EIP-712, a type hash is required. */
+    bytes32 constant VOTE_MESSAGE_TYPEHASH = keccak256(
```

Comment:
In line with other type hash names:
```suggestion
    bytes32 constant VOTEMESSAGE_TYPEHASH = keccak256(
```
(It's also what the [EIP-712 example implementation](https://github.com/ethereum/EIPs/blob/master/assets/eip-712/Example.sol#L23) does: `EIP712DOMAIN_TYPEHASH`)